.dotnet/tests/Samples/Images/Sample02_SimpleImageEdit.cs

```diff
@@ -14,26 +14,26 @@ public void Sample02_SimpleImageEdit()
             ImageClient client = new("dall-e-2", Environment.GetEnvironmentVariable("OpenAIClient_KEY"));
 
             string imagePath = Path.Combine("Assets", "edit_sample_image.png");
-            BinaryData imageBytes = BinaryData.FromBytes(File.ReadAllBytes(imagePath));
+            using FileStream inputImage = File.OpenRead(imagePath);
 
             string prompt = "An inflatable flamingo float in a pool";
 
             string maskPath = Path.Combine("Assets", "edit_sample_mask.png");
-            BinaryData maskBytes = BinaryData.FromBytes(File.ReadAllBytes(maskPath));
+            using Stream mask = File.OpenRead(maskPath);
 
             ImageEditOptions options = new()
             {
-                MaskBytes = maskBytes,
+                Mask = mask,
                 MaskFileName = "edit_sample_mask.png",
                 Size = ImageSize.Size1024x1024,
                 ResponseFormat = ImageResponseFormat.Bytes
             };
 
-            GeneratedImageCollection image = client.GenerateImageEdits(imageBytes, "edit_sample_image.png", prompt, 1, options);
-            BinaryData bytes = image[0].ImageBytes;
+            GeneratedImageCollection imageCollection = client.GenerateImageEdits(inputImage, prompt, 1, options);
```

Comment:
GeneratedImageCollection may need to implement IDisposable now -- making the Image a Stream is viral like that.  We should weigh this tradeoff -- but I think it's needed to avoid loading large file data into memory without being able to control that.

Reply:
If we end up with images too big for memory, how will customers consume them?

In contrast to the other API calls that are just `application/octet-stream` raw data, these images still come back as a JSON document with the image encoded as a "data:" URI. Do we have a mechanism to incrementally deserialize the overall JSON document such that the data stream would actually be a stream?

Concretely: a partial, in-progress image generation response might look like the following:
```json
{
  "created": 1707347031,
  "data": [{
            "revised_prompt": "...revised prompt here...",
            "url": "data:image/png;base64,Dhe3j4nh3n3h
```

Reply:
@trrwilson, I think it might be hard to answer this question well without a good sense of how real end-users to clients are using these APIs in applications they're building.  Is that data we might be able to get access to?  Are these desktop UIs, or server apps, e.g.?  What is the use-case for a collection of generated images, e.g.?

Reply:
Resolving comment since the experimental "Stream output" approach has been removed from the PR.